devel/ubertooth/Portfile

```diff
@@ -0,0 +1,74 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           conflicts_build 1.0
+PortGroup           github 1.0
+PortGroup           legacysupport 1.1
+
+github.setup        greatscottgadgets ubertooth 3e9de91f406ea13f4a5a76c3fd5e21c2a0375ad8
+version             2022-07-19
+revision            0
+categories          devel net security
+description         Software, firmware and hardware designs for Ubertooth.
+long_description    Project Ubertooth is an open source wireless development platform suitable \
+                    for Bluetooth experimentation. Ubertooth ships with a capable BLE (Bluetooth Smart) \
+                    sniffer and can sniff some data from Basic Rate (BR) Bluetooth Classic connections.
+license             GPL-2
+maintainers         {@barracuda156 gmail.com:vital.had} openmaintainer
+
+homepage            https://github.com/greatscottgadgets/ubertooth
+
+checksums           rmd160  3a3545670f44bcb3a1ce257bf5dd5da42fba1014 \
+                    sha256  3ad1dc36accd8b7f69fdf4489df93afcaa4d1e7bdd9448b24a23eacc525deec5 \
+                    size    1307090
+
+# clock_gettime
+legacysupport.newest_darwin_requires_legacy 15
+
+patchfiles          patch-python3.diff
+
+set python_branch   3.10
+set python_ver      [string map {. {}} ${python_branch}]
+configure.python    ${prefix}/bin/python${python_branch}
+
+depends_build-append \
+                    port:pkgconfig
+
+if {${build_arch} ni [list ppc ppc64]} {
+    depends_build-append \
+                    port:python${python_ver} \
+                    port:py${python_ver}-numpy \
+                    port:py${python_ver}-pyside \
+                    port:py${python_ver}-setuptools
```

Comment:
are you sure that some of these are not runtime dependencies as well? It seems unlikely to me that all of them are only needed at build-time

Reply:
You are likely right. I will look into it.

Reply:
`setuptools` appears to be only a build dependency. Also  upstream list `qtpy` as a dependency, is that not needed? Why did you decide to use `pyside` instaed of more recent Qt5 or PySide2?

Reply:
Sorry, missed these comments. Gimme a min.

Reply:
@reneeotten Hmm, because their documentation is inconsistent. Check here: https://ubertooth.readthedocs.io/en/latest/building_from_git.html
Compare to: https://ubertooth.readthedocs.io/en/latest/build_guide.html
Or git version uses different Python ports.

Reply:
I do not think we want to use `-pyside2`, it is mostly broken: https://ports.macports.org/port/py310-pyside2/details/